### ISSUE TITLE
BCL website certificate issue

### ISSUE THREAD

**Issue by macumber** [BuildingComponentLibrary/bcl-manifest#37]
@tijcolem @kflemin Currently there seems to be an issue with the certificate associated with https://bcl.nrel.gov/, all API queries are failing with ECONNRESET

**Comment by kflemin** [comment]
@macumber, try again today?  We had power outages at NREL yesterday...

**Comment by macumber** [comment]
Thanks @kflemin the power outage explains the issues yesterday.  However, I was still having trouble with a search that was working last week but seems to have stopped working, the following search returns 0 results when previously (last weekend) it was non-empty:

`https://bcl.nrel.gov/api/metasearch/%2A.xml?fq%5B%5D=bundle:nrel_measure&fq%5B%5D=tid:953&api_version=2.0`

When I search `https://bcl.nrel.gov/api/metasearch/%2A.xml?fq%5B%5D=bundle:nrel_measure&api_version=2.0` I see:

```
<taxonomy_term>
    <name>Envelope</name>
    <tid>953</tid>
    <count>82</count>
</taxonomy_term>
```

Did something change?  Is the search by tid no longer working?

**Comment by kflemin** [comment]
I made some updates to the BCL, so perhaps something got broken in the process. i'll look into it. Search by tid should work!

**Comment by kflemin** [comment]
@macumber, I think I fixed it. Would you try again and let me know if that looks correct? thanks!

**Comment by macumber** [comment]
Yup, that fixed it, thanks so much!  For my information, should we be updating this query to a more recent version or is the query I am using supported for the long term?

**Comment by kflemin** [comment]
The query you are using is the "legacy" API, but we don't have plans to stop supporting it.  It might not get as much attention as the new/stable API syntax, though. The stable API doesn't let you access by TID anymore, but you could do something like the following to retrieve the same information:

https://bcl.nrel.gov/api/metasearch/*.xml?fq=bundle:measure&fq=measure_tags:Envelope

You can see example queries in the new syntax here: https://bcl.nrel.gov/documentation#examples
